requestor/src/test/java/TestAgeRequests.java

```diff
@@ -0,0 +1,89 @@
+import ca.uhn.fhir.context.FhirContext;
+import de.rwth.imi.flare.api.FlareResource;
+import de.rwth.imi.flare.api.model.*;
+import de.rwth.imi.flare.api.model.mapping.MappingEntry;
+import de.rwth.imi.flare.requestor.FhirRequestor;
+import de.rwth.imi.flare.requestor.FhirSearchRequest;
+import de.rwth.imi.flare.requestor.SearchQueryStringBuilder;
+import org.junit.jupiter.api.Test;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.List;
+
+public class TestAgeRequests {
+    @Test
+    public void mainAgeTests() throws URISyntaxException {
+        /* Suggestion for test data birthdates (relative to 2022-11-14):
+        *   A: 1983-05-17
+        *   B: 1990-01-01
+        *   C: 1990-11-01
+        *   D: 2022-10-01
+        *   E: 2022-11-01
+        * */
+
+        ageSingeComparisonRequest(35.0, "a", Comparator.gt); //should find only A
```

Comment:
@bastianschaffer : your unit tests should run without external services such as a FHIR server.
Instead of the printFhirRequestResult you should use 
assert 
to check the search String against the expected result, which you define beforehand.
something similar to:
assertEquals("Patient?birthdate=lt1987-11-16",ageSingeComparisonRequest(35.0, "a", Comparator.gt));

Note that your expected date will change every day as it is calculated on the fly => ensure that your expected value  -as in 1987-11-16 is updated accordingly.